ucsb_ds_capstone_projects_2021/projects/calcofi1/update2.md

```diff
@@ -27,13 +27,13 @@ In our last progress update, we took a brief look at the visual relationship bet
 
 <img src="https://raw.githubusercontent.com/gavintieng/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/calcofi1/update2_exploratory.png" alt="drawing" />
```

Comment:
Markdown can embed images directly: https://jupyterbook.org/content/figures.html
You want to add the image `update2_exploratory.png` to the same directory and replace the HTML with `![some-text](update2_exploratory.png)`